nucypher/policy/collections.py

```diff
@@ -229,6 +247,36 @@ def __repr__(self):
         return f"{self.__class__.__name__}:{self.public_id()[:6]}"
 
 
+class DecentralizedTreasureMap(TreasureMap):
```

Comment:
Why `DecentralizedTreasureMap` and not `SignedTreasureMap`?

The prefix of `Decentralized` here doesn't really add much contextual information.

Reply:
Yeah, I like that.